.github/workflows/cicd.yml

```diff
@@ -12,55 +12,45 @@ jobs:
   dartdoc:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
-      - uses: actions/setup-java@v1
-        with:
-          java-version: '12.x'
+      - uses: actions/checkout@v2
```

Comment:
actually this is a dart-only packages; our scripts use the dart commands I believe for the checks. the github actions are for flutter right?

Reply:
You mean so that there isn't a diff when running the scripts locally and on the PR? Doesn't the analysis_options file ensure that there isn't any diff even if it goes through flutter? This cicd pipeline used some flutter commands before too, but maybe it was only to get dart.

Reply:
Better to keep the cicd and the script folder from before and just add the `flame_lint` package maybe?